src/commands/info.rs

```diff
@@ -133,15 +134,8 @@ pub fn execute(matches: &ArgMatches) -> Result<()> {
                     println!("    - {scope}");
                 }
             }
+            Ok(())
         }
-        Err(err) => {
-            println!("  (failure on authentication: {err})");
-        }
-    }
-
-    if errors {
-        Err(QuietExit(1).into())
-    } else {
-        Ok(())
+        Err(err) => Err(anyhow::anyhow!(err)),
```

Comment:
I would suggest replacing this entire match:
```
    let info = info_rv?;
    if let Some(ref user) = info.user {
        println!("  User: {}", user.email);
    }
    if let Some(ref auth) = info.auth {
        println!("  Scopes:");
        for scope in &auth.scopes {
            println!("    - {scope}");
        }
    }
    Ok(())
```

Reply:
Nice, yeah that looks much cleaner!